lib/sync/worker.js

```diff
@@ -80,4 +83,12 @@ QueueWorker.prototype.work = function() {
   });
 };
 
+/**
+ * Stop the queue worker
+ */
+QueueWorker.prototype.stop = function(cb){
```

Comment:
as this is synchronous, can leave out a callback. No need to return anything either. If function finishes, the caller knows it was marked as stopped.